neo/Persistence/LevelDB/LevelDBStore.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Neo.Cryptography.ECC;
+using Neo.IO.Caching;
+using Neo.IO.Data.LevelDB;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using System;
+using System.Reflection;
+
+namespace Neo.Persistence.LevelDB
+{
+    public class LevelDBStore : Store, IDisposable
+    {
+        private readonly DB db;
+
+        public LevelDBStore(string path)
+        {
+            this.db = DB.Open(path, new Options { CreateIfMissing = true });
+            if (db.TryGet(ReadOptions.Default, SliceBuilder.Begin(Prefixes.SYS_Version), out Slice value) && Version.TryParse(value.ToString(), out Version version) && version >= Version.Parse("3.0.0"))
```

Comment:
When you bumped the version back to 2.9.0 you broke loading the DB since this checks version >= 3.0.0
This needs to be changed to 2.9.0